docs/configuration.md

```diff
@@ -22,6 +22,7 @@ Property
 Property                                                                                    | Description                                                                                                                                                                               | Default value
 ------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------
 **envoy-control.envoy.snapshot.edsConnectionTimeout**                                       | Connection timeout for EDS clusters                                                                                                                                                       | 2s
+**envoy-control.envoy.snapshot.egress.commonHttp.idleTimeout**                              | Set idle timeout for all HTTP connections (HTTP/1 and HTTP/2)                                                                                                                             | 120s
```

Comment:
I see that this configuration doesn't look good now. We should create some task to refactor this. Here is my proposition we can talk about format later.
```yaml

envoy-control:
  envoy:
    snapshot:
      egress:
        http:
          protocol: HTTP2
          http2-configuration:
            specific-configuration
          idle-timeout:
       tcp: #in future
         some-configuration
```

Reply:
I made an issue for this.